.gitlab/image_deploy/docker_windows.yml

```diff
@@ -70,6 +70,33 @@ dev_branch_docker_hub-a7-windows:
     - docker run --rm -w C:\mnt -e AWS_NETWORKING=true -e SIGN_WINDOWS=true -v "$(Get-Location):C:\mnt" -v \\.\pipe\docker_engine:\\.\pipe\docker_engine 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_${Env:VARIANT}_x64:${Env:DATADOG_AGENT_WINBUILDIMAGES} powershell -C C:\mnt\ci-scripts\publish.ps1
     - If ($lastExitCode -ne "0") { throw "Previous command returned $lastExitCode" }
 
+dev_branch_docker_hub-a6-windows:
```

Comment:
This addition will conflict with the new way used in this PR: https://github.com/DataDog/datadog-agent/pull/8134